Documentation/README.md

```diff
@@ -1,17 +1,18 @@
-# Rook
+# Rook Documentation
 
-## Quickstart Guides
+## Getting Started
```

Comment:
i think this is a nice reorganization of the links on this page into categories that also removes any content duplication from the main readme at the root of the repo.